### PR TITLE
print the model parameters when train/eval the model

### PR DESCRIPTION
**Pull request by zhtmike**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: mindocr/models/builder.py [hunk 1]
```diff
@@ -75,4 +77,7 @@ def build_model(name_or_config: Union[str, dict], **kwargs):
     if "amp_level" in kwargs:
         auto_mixed_precision(network, amp_level=kwargs["amp_level"])
 
+    num_params = sum([param.size for param in network.get_parameters()])
+    _logger.info(f"Total number of model parameters: {num_params}")
```
Comment (hadipash) [inline mindocr/models/builder.py:81]: Would be nice to specify the number of trainable and non-trainable parameters as well.

**Review by hadipash — APPROVED** [review]

File: mindocr/models/builder.py (hunk 1 above)
Comment (SamitHuang) [reply mindocr/models/builder.py:82]: We may put this log info into the key info section  (=== ... ===) in train.py/eval.py for easy spotting.

Comment (zhtmike) [reply mindocr/models/builder.py]: done

Comment (hadipash) [reply mindocr/models/builder.py]: since logs are separated now, set `num_trainable_params` for training only?

Comment (zhtmike) [reply mindocr/models/builder.py]: I think evaluation script still needs this information, if user doesn't like to retrain but still need to spot this info

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**